source/features/open-issue-to-latest-comment.tsx

```diff
@@ -3,9 +3,14 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
+const selector = `
+	:is(.js-issue-row, .js-pinned-issue-list-item)
+	.Link--muted:is([aria-label$="comment"], [aria-label$="comments"])
+`;
+
 function init(): void {
-	for (const link of select.all('.js-issue-row a[aria-label*="comment"], .js-pinned-issue-list-item a[aria-label*="comment"]')) {
-		link.hash = '#partial-timeline';
+	for (const link of select.all<HTMLAnchorElement>(selector)) {
+		link.hash = '#issue-comment-box';
```

Comment:
I also changed the element because it seems that on ajax navigation `partial-timeline` does not appear. Also this `#issue-comment-box` element appears to exist specifically for the purpose.

Reply:
@busches want to check GHE if this exists?

Reply:
hmm strangely I'm seeing it now. I'll revert because `#issue-comment-box` has the side effect of showing a focus ring around the comment box, sometimes
```suggestion
		link.hash = '#partial-timeline';
```